src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1,10 +1,9 @@
 // File system related system calls
 use crate::interface;
-
-use super::fs_constants::*;
-use crate::safeposix::cage::{CAGE_TABLE, Cage, FileDescriptor::*, FileDesc, FdTable};
+use crate::interface::types::{FSData, StatData, Arg};
```

Comment:
you imported these via cage in another file?

Reply:
Oops -- missed those. Will fix that now.